services/minespace-web/common/components/PartyAppointmentTable.js

```diff
@@ -204,6 +208,7 @@ const mapDispatchToProps = (dispatch) =>
     {
       updatePartyRelationship,
       fetchPartyRelationships,
+      // getUserInfo,
```

Comment:
I'm wondering why this was removed?  Is this handled with in the new `/me` endpoint instead?